src/Drivers/pseudo.hpp

```diff
@@ -67,6 +80,17 @@ struct NonLocalPP
     // clang-format on
   }
 
+  // create VPs
+  void initialize_VPs(const ParticleSet& ions, const ParticleSet& elecs, const RealType Rmax_in)
+  {
+    if(VPs.size()) throw std::runtime_error("Can not create VPs again.\n");
+    // reserving space avoids the copy constructor of VirtualParticleSet which is not safe to use.
```

Comment:
So VirtualParticleSet is broken by design?

Reply:
As I explained above, ParticleSet is a bit broken.